netci.groovy

```diff
@@ -44,7 +44,7 @@ def addBuildStepsAndSetMachineAffinity(def job, String os, String configuration)
         shell("git submodule update --init --recursive");
         shell("./build.sh /p:Configuration=${configuration} ${loggingOptions}");
         smokeTestExcludes = "";
-        if (os == "OSX10.12") {
+        if (os == "Fedora24" || os == "OSX10.12") {
```

Comment:
We probably don't need this change in master.  Master is updated to build on Fedora28, now.

Reply:
Good call, removed.